packages/atom/test/slice.test.ts

```diff
@@ -216,7 +216,7 @@ describe('@bigtest/atom Slice', () => {
 
       expect(atom.get()).toEqual(['foo', 'bar']);
       slice.remove();
-      expect(atom.get()).toEqual([undefined, 'bar']); // TODO: shouldn't this remove the item entirely?!
+      expect(atom.get()).toEqual(['bar']);
```

Comment:
There is a case to be made for either behaviour here. The value of the slice will now have changed to `bar` which is a bit odd.